scripts/diamond_clean_up.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
This file is no longer with the `uninstall` operation added to the diamond plugin